boot.php

```diff
@@ -35,6 +35,13 @@
     }
 }
 
+// switch REDAXO to frontend mode before generating cache files
+rex_extension::register('PACKAGES_INCLUDED', function(rex_extension_point $ep) {
+    if (rex_be_controller::getCurrentPagePart(2) == 'generator') {
```

Comment:
kannst du diesen schalter nicht stattdessn direkt in der generator-page machen statt hier über nen EP?